main/generate.es6.js

```diff
@@ -27,11 +33,6 @@ const argv = require('yargs')
         default: '',
         coerce: bStr => bStr.split(',').filter(str => !!str).map(str => str.toLowerCase()),
       })
-      .option('from-confluence', {
```

Comment:
@jpmedley do you use this flag? See @foolip's PR-level comments on why it would be easier if we could remove it.

Reply:
Sorry I missed this. FWIW, I don't think I've ever used this flag.